CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Set `blockOwnerDeletion` to `true` in the owner references in Strimzi managed resources.
   Deleting the Strimzi custom resources will now by default wait for the deletion of all the owned Kubernetes resources.
+* Update quartz dependency to version 2.5.0 in order to fix [CVE-2023-39017](https://nvd.nist.gov/vuln/detail/cve-2023-39017)
```

Comment:
We usually do not keep regular dependency updated in the CHANGELOG. It makes it pretty hard to read and makes it easy to miss the really important things.